src/radiant.c

```diff
@@ -3015,8 +3060,13 @@ int radiant_get_fw_version( const radiant_dev_t* bd, const radiant_dest_t which,
 
 uint16_t radiant_get_sample_rate(const radiant_dev_t * bd) 
 {
+  
   if (!bd) return -1; 
-  return 2400; //as of aug 2023 we lowered the sampling rate from 3200 to 2400MHz
+  if(bd->rad_dateversion_int<=230)
```

Comment:
since 0.2.16, the board manager will support reading out the sample rate in MHz in register 0xf0.